app/views/layouts/application.html.erb

```diff
@@ -13,11 +13,10 @@
   <% end %>
   </title>
   <%= csrf_meta_tags %>
-  <%= stylesheet_link_tag 'application', media: 'all' %>
-  <%= stylesheet_link_tag 'print', media: 'print' %>
```

Comment:
We can either create a new pack like `frontend/packs/print.js` which can import style sheets required for print. Or we might find it best to do the media query. It will depend on what requirements we get from our components.